pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/ExcelParticipantExporter.java

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.dsm.model.elastic.export.tabular;
+
+import javax.servlet.ServletOutputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+import com.google.common.net.MediaType;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.streaming.SXSSFSheet;
+import org.apache.poi.xssf.streaming.SXSSFWorkbook;
+import spark.Response;
+
+public class ExcelParticipantExporter extends TabularParticipantExporter {
+    private final SXSSFWorkbook workbook = new SXSSFWorkbook(200);
```

Comment:
I would extract magic number `200` just to make easier for others who don't know the details of constructor of `SXSSFWorkbook`